templates/zerver/help/edit-or-delete-a-message.md

```diff
@@ -33,11 +33,10 @@ content.
     assuming that feature has not been
     [disabled by an organization administrator](/help/disable-message-edit-history).
 
-If you don't see the pencil (<i class="fa fa-pencil"></i>) icon, the message content
-can no longer be edited. You should see a file (<i class="fa fa-file-code-o"></i>)
-icon instead. Clicking the file icon will allow you to view the
-[Markdown source](/help/view-the-markdown-source-of-a-message) of the message, or
-[edit the topic](/help/rename-a-topic).
+If you don't see the **pencil** (<i class="fa fa-pencil"></i>) icon, the content
+of your message can no longer be edited. You will see the **move**
+(<i class="fa fa-arrows move_message_button edit_message_button"></i>)
```

Comment:
Let's drop the "You will..." part. It's more complicated, since the user might not have permissions to move the message either.